src/app/components/core/control-dispatcher/list-control-dispatcher/list-control-dispatcher.component.html

```diff
@@ -1,4 +1,24 @@
 <div class="container is-fluid">
+  <ng-container *ngIf="dispatcherServiceEnabled === false">
```

Comment:
This is not correct. what if dispatcher service is not installed.
this.dispatcherServiceEnabled is only being set only if service is installed.